controllers/user.js

```diff
@@ -41,6 +41,10 @@ exports.get_user = (req, res, next) => {
         res.item(result[0])
             .send();
     }
+    
+    function try () {
```

Comment:
Expected an identifier and instead saw 'try' (a reserved word).<br>'try' is defined but never used.